packages/server/graphql/mutations/helpers/notifications/SlackNotifier.ts

```diff
@@ -304,9 +405,22 @@ export const SlackNotifier: Notifier = {
 
   async endMeeting(dataLoader: DataLoaderWorker, meetingId: string, teamId: string) {
     const {meeting, team} = await loadMeetingTeam(dataLoader, meetingId, teamId)
+    const meetingResponses = await getTeamPromptResponsesByMeetingId(meetingId)
+    const standupResponses: Array<{user: User; response: TeamPromptResponse}> = []
+    Promise.allSettled(
+      meetingResponses.map(async (response) => {
+        const user = await dataLoader.get('users').load(response.userId)
```

Comment:
We _could_ do a single `loadMany` with all the userIds, but `DataLoader` is good at batching, and these userIds seem to already be in the dataloader cache when a meeting ends here anyway, so this seems like the best way to connect users and their responses.

Reply:
💯 if you wanted to make it a pure function you could use `loadNonNull` since user is guaranteed

```ts
const standupResponses = await Promise.all(
  meetingResponses.map(async (response) => {
    const user = await dataLoader.get('users').loadNonNull(response.userId)
    return {user, response}
  })
)
```